src/pal/inc/rt/palrt.h

```diff
@@ -1502,7 +1502,7 @@ typedef struct _DISPATCHER_CONTEXT {
     PRUNTIME_FUNCTION FunctionEntry;
     DWORD EstablisherFrame;
     DWORD TargetIp;
-    PCONTEXT CurrentRecord;
+    PKNONVOLATILE_CONTEXT CurrentRecord;
```

Comment:
A nit - could you please rename the CurrentRecord, which is a name that doesn't tell what it really represents to CurrentNonVolatileContextRecord or CurrentNonVolatileRegisters? Otherwise LGTM.